Installation/Schema/klant.contactmoment.schema.json

```diff
@@ -1,7 +1,7 @@
 {
     "$id": "https://klantenBundle.commonground.nu/klant.contactmoment.schema.json",
     "$schema": "https://docs.commongateway.nl/schemas/Entity.schema.json",
-    "version": "0.2",
+    "version": "0.1",
```

Comment:
You are going down a version here, this is intended?

Reply:
ah thanks, this isn't intended.